core/src/autogluon/core/searcher/bayesopt/autogluon/gp_multifidelity_searcher.py

```diff
@@ -104,90 +102,39 @@ def __init__(
             this many candidates received label feedback
         :param initial_scoring: Scoring function to rank initial candidates.
             Default: thompson_indep (independent Thompson sampling)
-        :param profiler: If given, HPO computations are profiled
         :param first_is_default: If true, the first result of get_config is the
             default config of hp_ranges
         :param debug_log: DebugLogPrinter for debug logging (optional)
+        :param profiler: If given, HPO computations are profiled
```

Comment:
cost_metric_name is missing from the :param list.

Reply:
OK